### ISSUE TITLE
[BUG] unable to compile `make fuzz_deposit-struct`

### ISSUE THREAD

**Issue by Daft-Wullie** [sigp/beacon-fuzz#69]
## Description
unable to compile the fuzz_deposit-struct in Beaconfuzz_v2

## Expected Behavior
the fuzzer should compile and execute against the specified target

## Actual Behavior
compiling the fuzzer fails with error[E0107]: wrong number of type arguments: expected 0, found 1

## Possible Fix
N/A

## Steps to Reproduce

1. run `make fuzz_deposit-struct`
2. fails to compile with following output    
```Compiling beaconfuzz_v2-fuzz v0.0.0 (/root/beacon-fuzz/beaconfuzz_v2/fuzz)
error[E0107]: wrong number of type arguments: expected 0, found 1
   --> fuzz_targets/struct_deposit.rs:142:32
    |
142 | fuzz_target!(|wrapper: Deposit<MainnetEthSpec>| {
    |                                ^^^^^^^^^^^^^^ unexpected type argument

error: aborting due to previous error

For more information about this error, try `rustc --explain E0107`.
error: could not compile `beaconfuzz_v2-fuzz`.

To learn more, run the command again with --verbose.
Error: failed to build fuzz script: "cargo" "build" "--manifest-path" "/root/beacon-fuzz/beaconfuzz_v2/fuzz/Cargo.toml" "--target" "x86_64-unknown-linux-gnu" "--release" "--bin" "struct_deposit"
make: *** [Makefile:79: fuzz_deposit-struct] Error 1 

```

![possiblebeaconfuzzbug](https://user-images.githubusercontent.com/37898858/92006358-ea23df80-ed44-11ea-925f-fae4574ac1cd.JPG)

## Your Environment
* Version/Commit used: latest
* Operating System and version: Ubuntu 20.04

**Comment by Daft-Wullie** [comment]
nvm, seems like this was actually fixed in commit 1d464e7 but  to perform my install i followed the readme that specifies the development branch and so did not have the updated target.

**Comment by Daft-Wullie** [comment]
fixed by commit 6619740
closing it.
